### PR TITLE
Add wait on removing playerdata

### PR DESCRIPTION
**Pull request by Kmack710**
**Describe Pull request**
Without the wait you cant use that players data at all on any other playerDropped events. Making you unable to get players names or citizenids for logs this 10 second wait gives scripts enough time to get the players info and use it before its gone.

If your PR is to fix an issue mention that issue here
 - Cant use any player data on log out like check job to remove from duty or get their name/citizenId for any logs 
 
**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? Yes
- Does your code fit the style guidelines? Yes
- Does your PR fit the contribution guidelines? Yes

### CODE REVIEW

**Review by uShifty — APPROVED** [review]
Looks good to me, I have noticed this aswell

**Comment by uShifty** [comment]
You will need to update your pull request to remove the trailing whitespaces on line 20 to ensure it passes the lint check

**Comment by FjamZoo** [comment]
I would recommend using a TimeOut instead of a createthread + timer, cleaner code and better functionality.

**Comment by GhzGarage** [comment]
image

File: server/events.lua [hunk 1]
```diff
@@ -8,15 +8,19 @@ AddEventHandler('chatMessage', function(_, _, message)
...
 
 AddEventHandler('playerDropped', function(reason)
-    local src = source
-    if not QBCore.Players[src] then return end
-    local Player = QBCore.Players[src]
-    TriggerEvent('qb-log:server:CreateLog', 'joinleave', 'Dropped', 'red', '**' .. GetPlayerName(src) .. '** (' .. Player.PlayerData.license .. ') left..' ..'\n **Reason:** ' .. reason)
-    Player.Functions.Save()
-    QBCore.Player_Buckets[Player.PlayerData.license] = nil
-    QBCore.Players[src] = nil
+    CreateThread(function()
```
Comment (BerkieBb) [inline server/events.lua:11]: Change this to SetTimeout and remove the wait and add `10000` after the `end` that closes the function to specify the timeout

**Review by BerkieBb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**